File: 800/59A_Word.py
cad = input()
minus = 0
mayus = 0

for c in cad:
    if c.islower():
        minus += 1
    elif c.isupper():
        mayus += 1

if mayus>minus:
    cad = cad.upper()
else:
    cad = cad.lower()
    
print(cad)    

File: 800/59A_Word.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    string cad; cin>>cad;
    int minus=0, mayus=0;
    for(char c: cad){
        if(islower(c)){
            minus++;
        }else if(isupper(c)){
            mayus++;
        }
    }
    if(mayus>minus){
        for(char &c : cad){
            c = toupper(c);
        }
    }else{
        for(char &c : cad){
            c = tolower(c);
        }
    }

    cout<<cad;
    return 0;
}